docs/modules/plugins/pages/plugin-azure-resource-manager.adoc

```diff
@@ -221,3 +221,51 @@ resourceGroups/sample-resource-group/providers/Microsoft.KeyVault/vaults/sample-
 ----
 When I delete Azure resource with identifier `resourceGroups/TEST-RG/providers/Microsoft.DataFactory/factories/TEST-ADF/linkedservices/test-datalake` using API version `2018-06-01`
 ----
+
+== Secrets Management Tools
+
+=== https://learn.microsoft.com/en-us/azure/key-vault/[Azure Key Vault]
+
+==== Configuration
+
+NOTE: The properties marked with *bold* are mandatory.
+
+[cols="1,3,1", options="header"]
+|===
+|Property
+|Description
+|Example
+|Default value
+
+|[subs=+quotes]`azure.key-vault.processor.enabled`
+|State of processor for properties
+|`true`
+|`false`
+
+|[subs=+quotes]`*azure.key-vault.name*`
+a|Azure Key Vault Name
+[NOTE]
+====
+To read more about authentication configuration see xref:ROOT:azure-authentication.adoc#_authentication[Authentication]
```

Comment:
I believe it's already described in this document above

Reply:
removed